frontend/app/components/inplace-edit/directives/edit-pane/edit-pane.directive.js

```diff
@@ -88,110 +88,33 @@ function inplaceEditorEditPane(EditableFieldsState, FocusHelper, $timeout) {
 inplaceEditorEditPane.$inject = ['EditableFieldsState', 'FocusHelper', '$timeout'];
 
 
-function InplaceEditorEditPaneController($scope, $element, $location, $timeout, $q, $rootScope,
-    WorkPackageService, EditableFieldsState, ApiHelper, NotificationsService) {
-
-  var showErrors = function() {
-    var errors  = EditableFieldsState.errors;
-    if (_.isEmpty(_.keys(errors))) {
-      return;
-    }
-    var errorMessages = _.flatten(_.map(errors), true);
-    NotificationsService.addError(I18n.t('js.label_validation_error'), errorMessages);
-  };
+function InplaceEditorEditPaneController($rootScope, $scope, $element, $location, $timeout,
+    EditableFieldsState, NotificationsService, inplaceEditStorage, inplaceEditMultiStorage) {
 
   var vm = this;
   var field = $scope.field;
+  var wpStore = inplaceEditMultiStorage.stores.workPackage;
 
   this.submit = function() {
-    EditableFieldsState.save().then(function() {
-      $location.hash(null);
-      $timeout(function() {
-        $element[0].scrollIntoView(false);
-      });
-    });
-  };
-
-  this.handleFailure = function(e, submit) {
-    afterError();
-    EditableFieldsState.errors = {
-      _common: ApiHelper.getErrorMessages(e)
```

Comment:
This is the section that handled API errors previously. I've re-introduced it in my PR.